### PR TITLE
Release 2.9.4-beta2

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description

Looking for the PR for the upcoming second beta of the v2.9.4 series? Well you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
--- No new or changed feature flags
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated
--- No new metrics

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.9.4-beta2": [
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Fixed] Prevent styles from high-contrast theme from leaking - #12952",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Removed high-contrast specific hover effects from light and dark theme - #12952",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Merge head is correctly detected after merge conflicts resolved externally - #12881",
+      "[Fixed] Removed glitches when diffs are shown for the first time - #12903",
+      "[Improved] Remove gaps in the commit message container revealing a different background - #12900. Thanks @haykam821!",
+      "[Improved] Update \"remove repository\" dialog - #12497. Thanks @fonsp!",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Improved] Be more specific about what will get removed in the \"remove repository\" dialog - #12497. Thanks @fonsp!",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+      "[Improved] Update grammar for \"repo deletion\" and \"couldn't find pull request\" error messages - #12902. Thanks @patrykmichalik!",
+      "[Improved] Add host key verification to auth issue list - #12894. Thanks @thsmdt!"
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Improved] Show confirmation dialog for SSH host key verification - #12894. Thanks @thsmdt!"
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.9.4-beta2": [
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Fixed] Prevent styles from high-contrast theme from leaking - #12952",
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Merge head is correctly detected after merge conflicts resolved externally - #12881",
```
Comment (niik) [inline changelog.json]:
```suggestion
      "[Fixed] Merge flow continues after merge conflicts are resolved with exterior conflict tool - #12881",
```

**Review by niik — CHANGES_REQUESTED** [review]
A few suggestions for the release notes

File: changelog.json (hunk 2 above)
Comment (niik) [reply changelog.json]: @tidy-dev Just curious if there was a reason this suggestion didn't make it. More than happy to take another swing at it, it sure got wordy, but what we've got now isn't great

Comment (tidy-dev) [reply changelog.json]: I thought I accepted all of them. Sorry- must have scrolled passed it.

**Review by niik — DISMISSED** [review]

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.9.4-beta2": [
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Fixed] Removed high-contrast specific hover effects from light and dark theme - #12952",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Remove high-contrast specific hover effects from light and dark theme - #12952",
```

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,16 @@
...
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Merge flow continues after merge conflicts are resolved with exterior conflict tool - #12881",
+      "[Fixed] Removed glitches when diffs are shown for the first time - #12903",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Remove glitches when diffs are shown for the first time - #12903",
```

File: changelog.json [hunk 7]
```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.9.4-beta2": [
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Fixed] Removed high-contrast specific hover effects from light and dark theme - #12952",
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Merge flow continues after merge conflicts are resolved with exterior conflict tool - #12881",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Continue merge flow after merge conflicts are resolved with exterior conflict tool - #12881",
```

File: changelog.json [hunk 8]
```diff
@@ -1,5 +1,16 @@
...
+      "[Fixed] Removed glitches when diffs are shown for the first time - #12903",
+      "[Improved] Remove gaps in the commit message container revealing a different background - #12900. Thanks @haykam821!",
+      "[Improved] Be more specific about what will get removed in the \"remove repository\" dialog - #12497. Thanks @fonsp!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Add clearer verbiage in the \"remove repository\" dialog - #12497. Thanks @fonsp!",
```

**Review by billygriffin — COMMENTED** [review]
Couple minor suggestions!

**Review by billygriffin — APPROVED** [review]

**Pull request merged**